modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/ApplicationUI.macro

```diff
@@ -0,0 +1,33 @@
+definition {
+
+	macro changePublicationStatusViaUI {
```

Comment:
In the file we already point that we deal with UI, so maybe it's not necessary to have it in method. I'd put it as `changePublicationStatus`. This way if method's called it will be `ApplicationUI.changePublicationStatus()`

Reply:
How about update the filename to HeadlessBuilder.macro? cc @magjed4289 
![APIBuilder](https://github.com/liferay-headless/liferay-portal/assets/1308942/8a0c63e5-5f1a-41d0-a4c1-bb36bf5667bd)

Reply:
maybe APIBuilderUI.macro is better than HeadlessBuilder.macro

Reply:
Headless Builder is the name of the module, and API Builder is the name of the component. We need to decide which one we're sticking to and follow the pattern.

Reply:
Oh, I want to say APIBuilder.macro in my previous comment :D, then we can keep this, thanks!